NEW_TENANT.md

```diff
@@ -8,21 +8,39 @@ Creating a new tenant consists of two things:
 1. Create a new tenant
 2. Add a plan to the tenant
 
-There is two scripts (akvo.lumen.admin.add-tenant & akvo.lumen.admin-new-plan)
+There is two scripts ([akvo.lumen.admin.add-tenant](https://github.com/akvo/akvo-lumen/blob/master/backend/src/akvo/lumen/admin/add_tenant.clj) & [akvo.lumen.admin-new-plan](https://github.com/akvo/akvo-lumen/blob/master/backend/src/akvo/lumen/admin/new_plan.clj))
 for this. Both of these scripts needs inputs (secrets & configs). The scripts
 includes comments on what env vars are needed and where to find them. Since
 there is issues running the scripts on a Mac (because of the encryption library
 used) we run the scripts our docker container.
```

Comment:
... `in` our docker container 👀

Reply:
@tangrammer follow the instructions ... that is: `docker-compose exec [...]`